CodingAChat/Scenes/Edit user details/EditDetailsViewController.swift

```diff
@@ -71,6 +73,31 @@ class EditDetailsViewController: UIViewController {
         
     }
     
+    @IBAction func logout(_ sender: Any) {
+        let firebaseAuth = Auth.auth()
```

Comment:
We should logout in coordinator, not in view controller, due to view controller should manage only UI